### PR TITLE
Tests files upload formatting

### PR DESCRIPTION
**Pull request by anandsaini024**
Thanks for contributing. If this is your first time,
make sure to read the [contributing guide.](https://github.com/mne-tools/mne-python/blob/main/CONTRIBUTING.md)

PR Description
--------------

Changed the file uploading format. 
1. network_sample_output.m : added correct path to upload the files and save files in: tests/data/
2. test_iclabel_net: used "files" to upload the input files  as suggested by adam.

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/mne-tools/mne-connectivity/blob/main/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"

### CODE REVIEW

**Comment by mscheltienne** [comment]
Maybe put the matlab script to generate the .mat files used in `test_network_outputs` as a code-block comment in the [docstring of this function](https://github.com/jacobf18/iclabel-python/blob/49153bb7d1862e9479c0797660dc91c3675078c8/mne_icalabel/tests/test_iclabel_net.py#L77-L81). I don't see the need to have this short script as a separate file.
If you feel like it's too long for a docstring, then let's move it to `/mne_icalabel/tests/data`, IMO, `assets` should only contain large files required by `mne_icalabel` to work.

File: mne_icalabel/tests/test_iclabel_net.py [hunk 1]
```diff
@@ -18,12 +18,20 @@
 # load in test data for features from original Matlab ICLabel
 ica_file_path = str(files("mne_icalabel.tests").joinpath("data/eeglab_ica.set"))
 ica_raw_file_path = str(files("mne_icalabel.tests").joinpath("data/eeglab_ica_raw.mat"))
-torch_iclabel_path = Path(__file__).parent.parent / 'assets' / 'iclabelNet.pt'
-matconvnet_iclabel_path = Path(__file__).parent / 'data' / 'netICL.mat'
-mat_image_features_path = Path(__file__).parent / 'data' / 'matlab_images.mat'
-mat_psds_features_path = Path(__file__).parent / 'data' / 'matlab_psds.mat'
-mat_autocorr_features_path = Path(__file__).parent / 'data' / 'matlab_autocorrs.mat'
-mat_labels_file_path = Path(__file__).parent / 'data' / 'matlab_labels.mat'
+torch_iclabel_path = Path(__file__).parent.parent / "assets" / "iclabelNet.pt"
```
Comment (adam2392) [inline mne_icalabel/tests/test_iclabel_net.py:21]: this can use `files` as well.

Comment (mscheltienne) [reply mne_icalabel/tests/test_iclabel_net.py]: done

**Comment by mscheltienne** [comment]
It should be good to go now. Let's merge both PRs if happy, and I'll clean up a bit more after. I should have more time to allocate to this project this weekend and at the beginning of next week.

**Comment by adam2392** [comment]
Thanks @mscheltienne  and @anandsaini024 !

**Pull request merged**